### DISCUSSION TITLE
Scheduling bunch

### DISCUSSION THREAD

**Discussion by b-nln** [ttscoff/bunch-docs#245] [Q&A]
Hi,

I have a bunch, which should open weekdays at 7:55am and close at 6:30pm. 

I defined it as:

```text
---
title: email
open on: M T W Th F 7:55am
close on: M T W Th F 6:30pm
---
```

Sometimes it works, but most of the times it doesn't. I tried using `MTWThF 7:55am`, but it is same behavior. 

For example, it is now scheduled to run next Wednesday. Why  Wednesday? Why not Monday? 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/77802586/184459067-abf95673-8e80-4700-9ca1-c5cfc7f04ce7.png">

Looking at [original discussion](https://github.com/ttscoff/bunch/discussions/38) where idea was implemented, suggests that I'm doing this mostly right... but something is wrong?

Right now I re-wrote scheduling to look like this, will see how it works:

```
open on: MTWF 7:55am, Th 7:55am
close on: MTWF 6:30pm, Th 6:30pm
```

At least it now shows that it plans to run this bunch on Monday... Is there an issue with Thursday parsing? Perhaps, maybe give Thursday single-letter allies, `H`? On other hand, why not use crontab-based days as digits: 0-Sunday, 1-Monday, ... , 6-Saturday, 7-Sunday.

Any help would be appreciated.

**Comment by ttscoff** [comment]
The scheduling behavior has been a bit flaky for a few users. I don't 
have an immediate explanation for this particular issue. It is 
interesting that separating Thursday out caused it to work, though, and 
I'll double check the parsing on that.

-Brett

**Comment by b-nln** [comment]
After awhile, I'm sad to report, that what I though would be a workaround doesn't actually works. Sometimes after changing the bunch file it will re-read it and will open once or twice, and then will get stuck opening only once a week.
